packages/api/src/beacon/routes/proof.ts

```diff
@@ -45,7 +46,7 @@ export function getDefinitions(_config: ChainForkConfig): RouteDefinitions<Endpo
       url: "/eth/v0/beacon/proof/state/{state_id}",
       method: "GET",
       req: {
-        writeReq: ({stateId, descriptor}) => ({params: {state_id: stateId}, query: {format: toHexString(descriptor)}}),
+        writeReq: ({stateId, descriptor}) => ({params: {state_id: stateId}, query: {format: toHex(descriptor)}}),
```

Comment:
We can't use this here, otherwise the api package is no longer browser compatible, we need a switchable implementation, there is also another issue on ssz repo https://github.com/ChainSafe/ssz/issues/283.

Reply:
will switch to using ssz `toHexString()` for `api` for now

Reply:
need to work on this first https://github.com/ChainSafe/lodestar/pull/7060/files#diff-4e7247fcac06ac1a69d0886c9358f91dda8c0ad54bfa00b735517f313b11838fR9